demonstrations/tutorial_testing_symmetry.py

```diff
@@ -0,0 +1,459 @@
+r"""
+Testing for symmetry with quantum computers
+-------------------------------------------
```

Comment:
```suggestion
Testing for symmetry with quantum computers
===========================================
```

we tend to use the approach of `===` for the page title, and `---` for section titles (you can use other punctuation marks, such as `~~~`, etc., for sub(sub)sections -- ReST always assumes the order of punctuation determines the heading hierarchy).

This will ensure the sections appear in the right hand side ToC :)